documentation/src/main/java/org/hibernate/userguide/model/Person.java

```diff
@@ -42,155 +43,146 @@
-            "   pr.version AS \"pr.version\", " +
-            "   ph.id AS \"ph.id\", " +
-            "   ph.person_id AS \"ph.person_id\", " +
-            "   ph.phone_number AS \"ph.number\", " +
-            "   ph.phone_type AS \"ph.type\" " +
-            "FROM Person pr " +
-            "JOIN Phone ph ON pr.id = ph.person_id " +
-            "WHERE pr.name LIKE :name",
-        resultSetMapping = "person_with_phones"
-    )
-    //end::sql-entity-associations-NamedNativeQuery-example[]
-})
-@SqlResultSetMappings({
-    //tag::sql-entity-associations-NamedNativeQuery-example[]
-     @SqlResultSetMapping(
-         name = "person_with_phones",
-         entities = {
-             @EntityResult(
-                 entityClass = Person.class,
-                 fields = {
-                     @FieldResult( name = "id", column = "pr.id" ),
-                     @FieldResult( name = "name", column = "pr.name" ),
-                     @FieldResult( name = "nickName", column = "pr.nickName" ),
-                     @FieldResult( name = "address", column = "pr.address" ),
-                     @FieldResult( name = "createdOn", column = "pr.createdOn" ),
-                     @FieldResult( name = "version", column = "pr.version" ),
-                 }
-             ),
-             @EntityResult(
-                 entityClass = Phone.class,
-                 fields = {
-                     @FieldResult( name = "id", column = "ph.id" ),
-                     @FieldResult( name = "person", column = "ph.person_id" ),
-                     @FieldResult( name = "number", column = "ph.number" ),
-                     @FieldResult( name = "type", column = "ph.type" ),
-                 }
-             )
-         }
-     ),
-    //end::sql-entity-associations-NamedNativeQuery-example[]
-    //tag::sql-multiple-scalar-values-dto-NamedNativeQuery-example[]
-    @SqlResultSetMapping(
-        name = "name_and_nickName_dto",
-        classes = @ConstructorResult(
-            targetClass = PersonNames.class,
-            columns = {
-                @ColumnResult(name = "name"),
-                @ColumnResult(name = "nickName")
-            }
-        )
-    )
-    //end::sql-multiple-scalar-values-dto-NamedNativeQuery-example[]
-})
-//tag::hql-examples-domain-model-example[]
-@NamedQueries({
-    //tag::jpql-api-named-query-example[]
-    @NamedQuery(
-        name = "get_person_by_name",
-        query = "select p from Person p where name = :name"
-    )
-    //end::jpql-api-named-query-example[]
-    ,
-    // tag::jpa-read-only-entities-native-example[]
-    @NamedQuery(
-        name = "get_read_only_person_by_name",
-        query = "select p from Person p where name = :name",
-        hints = {
-            @QueryHint(
-                name = "org.hibernate.readOnly",
-                value = "true"
-            )
+//tag::sql-scalar-NamedNativeQuery-example[]
+@NamedNativeQuery(
+    name = "find_person_name",
+    query =
+        "SELECT name " +
```

Comment:
(Not a fan of capitalizing the keywords)*

Reply:
Neither am I, but that's not new. That's just how it was.

Reply:
Sure, but its stuff you explicitly changed already.  Why not just fix that at the same time?

Reply:
Well I think it's an all-or-nothing thing. There's actually rather a lot of example code there. I could go through and lowercase everything, but it will take me some time I don't really have _right now_. I can do it later when I go have time, or someone else can do it later. Here I was much more focussed one the organization/flow of the text.

Reply:
Oh, excuse me! I thought the existing code was using uppercase keywords everywhere, not just in that one code example.

Fixed. I've lowercased them.